pkg/controller/volume/attachdetach/util/util.go

```diff
@@ -377,14 +377,16 @@ func isCSIMigrationSupportedOnNode(nodeName types.NodeName, spec *volume.Spec, v
 		if err != nil {
 			return isMigratedOnNode, err
 		}
+
 		for _, driver := range csiNode.Spec.Drivers {
 			if driver.Name == driverName {
 				installed = true
 				break
 			}
 		}
 		if !installed {
-			return true, fmt.Errorf("in-tree plugin %s is migrated on node %s but driver %s is not installed", pluginName, string(nodeName), driverName)
+			klog.V(4).Infof("in-tree plugin %s is migrated on node %s but driver %s is not installed", pluginName, string(nodeName), driverName)
```

Comment:
I think we can just remove the entire section from L374. It also avoids one more potential error at L376.